packages/ketcher-react/src/icons/index.tsx

```diff
@@ -149,8 +149,102 @@ import DropdownIndicatorIcon from './files/dropdown-indicator.svg'
 import ChevronIcon from './files/chevron.svg'
 import FullscreenEnterIcon from './files/fullscreen-enter.svg'
 import FullscreenExitIcon from './files/fullscreen-exit.svg'
+import α from './files/α.svg'
+import β from './files/β.svg'
+import γ from './files/γ.svg'
+import δ from './files/δ.svg'
+import ε from './files/ε.svg'
+import ζ from './files/ζ.svg'
+import η from './files/η.svg'
+import θ from './files/θ.svg'
+import ι from './files/ι.svg'
+import κ from './files/κ.svg'
+import λ from './files/λ.svg'
+import μ from './files/μ.svg'
+import ν from './files/ν.svg'
+import ξ from './files/ξ.svg'
+import ο from './files/ο.svg'
+import π from './files/π.svg'
+import ρ from './files/ρ.svg'
+import σ from './files/σ.svg'
+import τ from './files/τ.svg'
+import υ from './files/υ.svg'
+import φ from './files/φ.svg'
+import χ from './files/χ.svg'
+import ψ from './files/ψ.svg'
+import ω from './files/ω.svg'
+import DegreeCelseus from './files/degree-celseus.svg'
```

Comment:
let's name icons according to convention, e.g. DegreeCelseusIcon
for symbols (not named icons) it's ok